lib/install/management/appliance.go

```diff
@@ -678,31 +676,51 @@ func (d *Dispatcher) CheckDockerAPI(conf *config.VirtualContainerHostConfigSpec,
 
 		// appliance is configured for tlsverify, but we don't have a client certificate
 		if len(conf.CertificateAuthorities) > 0 {
-			log.Debug("Loading CAs for client auth")
-			pool := x509.NewCertPool()
-			if !pool.AppendCertsFromPEM(conf.CertificateAuthorities) {
-				log.Debug("Unable to load CAs in config, if any")
-			}
+			// if tlsverify was configured at all then we must verify the remote
+			tr.TLSClientConfig.InsecureSkipVerify = false
 
-			// tr.TLSClientConfig.ClientCAs = pool
-			tr.TLSClientConfig.RootCAs = pool
+			func() {
```

Comment:
why do we have this? I don't see any defer's here so I guess we could remove the func(){ / {() parts, no?